mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -166,6 +166,16 @@ def create_registered_model(self, name, tags=None, description=None):
             created in the backend.
         """
         _validate_model_name(name)
+
+        # In case if under the hood of `store` is a GO implementation,
+        # it is impossible to use Pythin dynamic mocking.
+        # Tag with key="mock.time.time.fa4bcce6c7b1b57d16ff01c82504b18b.tag"
```

Comment:
What necessitates us to use "mock.time.time.fa4bcce6c7b1b57d16ff01c82504b18b.tag"? Could we use more human-readable name?

Reply:
I think we can make it a unique tag even using a human-readable string like `mock.go.testing.tag`?